### PR TITLE
Add ability to display system information for plugins

### PR DESCRIPTION
**Pull request by iliajie**
Related https://github.com/virtualmin/virtualmin-nginx/pull/72.

### CODE REVIEW

File: collect-lib.pl [hunk 1]
```diff
@@ -172,6 +172,13 @@ sub collect_system_info
 			}
 		}
 	}
+foreach my $f (split(" ", $config{'plugins'})) {
```
Comment (jcameron) [inline collect-lib.pl:175]: There's an @plugins global array for this, or @feature_plugins

File: collect-lib.pl [hunk 2]
```diff
@@ -172,6 +172,13 @@ sub collect_system_info
...
+	if (&plugin_defined($f, "feature_sysinfo")) {
+		$f =~ s/-/_/g;
+		my $ifunc = "${f}::feature_sysinfo";
```
Comment (jcameron) [inline collect-lib.pl:178]: There's a plugin_call function that does the module to package mapping

File: collect-lib.pl (hunk 1 above)
Comment (iliajie) [reply collect-lib.pl]: Thanks! Fixed now!

File: collect-lib.pl (hunk 2 above)
Comment (iliajie) [reply collect-lib.pl]: Oh, that's right! Great! Thanks! Fixed now!

**Pull request merged**